### DISCUSSION TITLE
Looking for a US based developer collaboration!

### DISCUSSION THREAD

**Discussion by DonJ92** [community/community#132758] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

We can make more efficient service or more funds in Github.

Thank you for considering my proposal. I look forward to the possibility of discussing how my background, skills, and enthusiasms align with your thoughts.

Please feel free email to me.

[redacted by moderator]

**Comment by ebndev** [comment]
Hi @DonJ92, thanks for being a part of the GitHub Community!

Unfortunately, we currently do not allow self-promotion or advertising on the Community Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.
